relayer/create_ibc_channel.go

```diff
@@ -34,15 +44,48 @@ func (r *Relayer) CreateIBCChannel(override bool, logFileOption utils.CommandOpt
 		return ConnectionChannels{}, err
 	}
 
+	// We always pass override otherwise this commands hangs if there are too many clients
```

Comment:
what the difference with previous version?
it worked without hanging, and without `override=true`

Reply:
it did only if you run it against local hub without thousands of ibc clients. if you run it against froopyland it hangs. the difference i the data on the hub which makes this command hangs as it searches first to see if an ibc client already exists on the hub and when it needs to iterates thousands of clients and consensus states it takes forever.